l10n_es_aeat_sii_oca/__manifest__.py

```diff
@@ -53,6 +54,7 @@
         "views/account_fiscal_position_view.xml",
         "views/res_partner_views.xml",
         "views/aeat_sii_tax_agency_view.xml",
+        "data/aeat_sii_queue_job.xml",
```

Comment:
What is the reason to add it?

Reply:
In v14, the `@job` decorator from `queue_job` has been deprecated and, instead, the functions and channels should be declared like this. See https://github.com/OCA/queue/tree/14.0/queue_job#developers for details.